README.md

```diff
@@ -99,149 +144,169 @@ This Dockerfile currently contains all installed C libraries and the Python wrap
+Exact definition of these parameters are described in file ***include/ccl_cls.h***. Usually you can use simplified versions of this function, namely **`ccl_cl_tracer_number_counts_new`, `ccl_cl_tracer_number_counts_simple_new`, `ccl_cl_tracer_lensing_new`** or **`ccl_cl_tracer_lensing_simple_new`**. Two most simplified versions (one for number counts and one for shear) take parameters:
 ````c
-CCL_ClTracer *ccl_cl_tracer_number_counts_simple_new(ccl_cosmology *cosmo, int nz_n,double *z_n,
-                                                     double *n, int nz_b,double *z_b,double *b);
-CCL_ClTracer *ccl_cl_tracer_lensing_simple_new(ccl_cosmology *cosmo, int nz_n,double *z_n,double *n);
-
+CCL_ClTracer *ccl_cl_tracer_number_counts_simple_new(ccl_cosmology *cosmo,
+						     int nz_n,double *z_n,double *n,
+                             int nz_b,double *z_b,double *b, int * status);
+CCL_ClTracer *ccl_cl_tracer_lensing_simple_new(ccl_cosmology *cosmo,
+					       int nz_n,double *z_n,double *n, int * status);
 ````
-where **nz_n** is dimension of arrays **z_n** and **n**. **z_n** and **n** are arrays for the number count of objects per redshift interval (arbitrary normalization - renormalized inside). **nz_b, z_b** and **b** are the same for the clustering bias.
-With initialized tracers you can compute limber power spectrum with **ccl_angular_cl**
+where `nz_n` is dimension (number of bins) of arrays `z_n` and `n`. `z_n` and `n` are arrays for the number count of objects per redshift interval (arbitrary normalization - renormalized inside). `nz_b`, `z_b` and `b` are the same for the clustering bias.
+
+With initialized tracers you can compute limber power spectrum with **`ccl_angular_cl`**
 ````c
-double ccl_angular_cl(ccl_cosmology *cosmo,int l,CCL_ClTracer *clt1,CCL_ClTracer *clt2);
+double ccl_angular_cl(ccl_cosmology *cosmo,int l,CCL_ClTracer *clt1,CCL_ClTracer *clt2, int * status);
 ````
-After you are done working with tracers, you should free its work space by **ccl_cl_tracer_free**
+After you are done working with tracers, you should free its work space by **`ccl_cl_tracer_free`**
 ````c
 void ccl_cl_tracer_free(CCL_ClTracer *clt);
 ````
 
 ### Halo mass function
-The halo mass function *dN/dM* can be obtained by function **ccl_massfunc**
+The halo mass function *dN/dM* can be obtained by function **`ccl_massfunc`**
 ````c
-double ccl_massfunc(ccl_cosmology * cosmo, double halo_mass, double redshift)
+double ccl_massfunc(ccl_cosmology * cosmo, double smooth_mass, double a, double odelta, int * status);
 ````
-where **halo_mass** is mass smoothing scale (in units of *M_sun/h*. For more details (or other functions like *sigma_M*) see *include/ccl_massfunc.h* and *src/mass_func.c*.
+where `smooth_mass` is mass smoothing scale (in units of *M_sun*) and `odelta` is choice of Delta. For more details (or other functions like **`sigma_M`**) see ***include/ccl_massfunc.h*** and ***src/mass_func.c***.
```

Comment:
Units have changed? @villarrealas can you verify this?

Reply:
I changed this based on information in ***include/ccl_massfunc.h***

Reply:
That is correct - I must have missed that I hadn't changed things in the readme. ccl_massfunc uses units of M_sun consistently throughout.